.npmignore

```diff
@@ -9,10 +9,12 @@ CONTRIBUTING.md
 appveyor.yml
 babel.config.js
 dist/test
+dist/typings
```

Comment:
Why are we not publishing the types?
If we want to produce embark as a lib, the types are helpful.

Reply:
That is a good question, usually TS generate those but I am not sure it is possible with babel, something to investigate.

I would say ignore my comments until we found a way to generate .d.ts file

Reply:
There is an option in ts config:
`"declaration": true,` but because we don't emit the files this is not possible.

Something to investigate but unsure would be https://www.npmjs.com/package/dts-bundle

Reply:
Maybe it's possible to emit only the bundled declaration. I can investigate, but I think making that change to the build should land in another PR.

Reply:
Probably a good idea to tackle in a new PR, though let's not forget about it :)